integration-tests/mllp/src/main/java/org/apache/camel/quarkus/component/mllp/it/MllpResource.java

```diff
@@ -55,4 +59,21 @@ public void sendInvalidMessageToMllp(String message) throws Exception {
 
         mockEndpoint.assertIsSatisfied(5000);
     }
+
+    @Path("/charset/default")
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getDefaultCharset() {
+        return MllpComponent.getDefaultCharset().name();
+    }
+
+    @Path("/charset/msh18")
+    @POST
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getWithCharsetFromMsh18(String message) {
+        String mlpHostPort = String.format("mllp:%s:%d", MllpRoutes.MLLP_HOST, MllpRoutes.MLLP_PORT);
```

Comment:
neat typo time permitting "mlpHostPort" :)

Reply:
Well spotted :eyes:!

I'll fix it. I won't sleep tonight knowing that I missed out an 'l' :stuck_out_tongue: